reference/orders.v3.yml

```diff
@@ -2732,54 +2872,70 @@ components:
           type: string
           description: |
             Namespace for the metafield, for organizational purposes.
-          minLength: 1
-          maxLength: 64
+          enum:
+            - 'minLength: 1'
+            - 'maxLength: 64'
```

Comment:
this field can hold any data, there should not be an enum here

Reply:
Thanks @donald-nguyen-bc for the feedback. Much appreciated! 😄